### DISCUSSION TITLE
Stale bot is on fire 🔥, should we increase the number of days?

### DISCUSSION THREAD

**Discussion by dhruvmanila** [TheAlgorithms/Python#8576] [Team Posts]
More than 300 pull requests have been marked stale and more than 50 have been closed. Should we increase the number of days for `days until stale` and `days until close`?

**Comment by dhruvmanila** [comment]
@cclauss @poyea

**Comment by cclauss** [comment]
My sense is that `limitPerRun: 30` is too high.  What about 5 or 7 instead?

**Comment by dhruvmanila** [comment]
That attribute is just how many PR + issues the bot will handle at a time, decreasing it will just make the bot take longer to handle all PRs/issues but not much. The bot will still be able to go through all the PRs/issues in a couple of days, marking/closing them as per the number of days. The main thing is how many days makes a PR stale and then how many days to close the PR.

**Comment by cclauss** [comment]
I take `limitPerRun` to be a `batch size` parameter.  I believe that it is reasonable maintainers have 5 to 7 new PRs per day to review.  I have been looking at all the PRs that stalebot has flagged so far and I do not see many that need to be saved.

**Comment by dhruvmanila** [comment]
Alright, let's set the `limitPerRun` attribute to 5. Also, I see a couple of branches that seem to be either stale or the PR related to that branch is closed. 

It is possible for the bot to delete the merged branch if it is from this repository.

**Comment by cclauss** [comment]
Let's get the open PRs down to a reasonable size before pruning branches.
